proposals/stage-0/SECURITY_REPORTING/README.md

```diff
@@ -0,0 +1,60 @@
+# My New Proposal
+>  Stage 0
+
+## Champion
+
+Marcin Hoppe (@marcin_hoppe)
+
+## Description
+
+Currently OpenJS Foundation projects do not have consistently documented standards for responsible reporting and disclosure of security vulnerabilities. This proposal aligns OpenJS Foundation projects with industry best practices in this space.
+
+See also:
+- [Security reporting in OpenJS Foundation projects](https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f)
+- [Guidelines for responsible reporting and disclosure of security vulnerabilities](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/security-guidelines.md)
+
+## Required Resources
+
+- Generic security policy template that projects can easily adopt
+- Optionally: vulnerability disclosure platform (VDP)
+
+## Who would be responsible?
+
+The responsibility for adoption of security reporting standards would be jointly owned by the projects and the CPC.
+
+## How would success be measured?
+
+- 100% of Impact, Growth, and At-Large projects have:
```

Comment:
Just wondering, do you have stats about the current status?

Reply:
@molant Yes, I did a bit of analysis a few weeks ago. It's referenced in the opening section of this proposal: https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f.

Reply:
I would not target all at-large. Some of those has very little activity happening, and adding a security process would be daunting. Maybe

- all of Impact and Growth projects
- the majority of at-large projects

Reply:
@mcollina I made the rules less strict for At-Large projects: https://github.com/openjs-foundation/cross-project-council/pull/489/files/#diff-c72fc357c4747ff075cec34a1d107ae7R28.

Do you think this could work?